src/webgl/p5.Camera.js

```diff
@@ -127,9 +127,29 @@ p5.prototype.camera = function (...args) {
  * vertical field of view, aspect ratio (usually width/height), and near and far
  * clipping planes.
  *
- * If no parameters are given, the following default is used:
- * perspective(PI/3, width/height, eyeZ/10, eyeZ*10),
- * where eyeZ is equal to ((height/2) / tan(PI/6)).
+ * If no parameters are given, the default values are used as:
+ *
+ * fov- The default field of view for the camera is such that the full height of renderer is visible when it is positioned at a default distance of 800 units from the camera.
+ *
+ * aspect- The default aspect ratio is the ratio of renderer's width to renderer's height.
+ *
+ * near - The default value for the near clipping plane is 0.1 times the default distance from the camera to the point it is looking at i.e 800.
+ *
+ * far - The default value for the far clipping plane is 10 times the default distance from the camera to the point it is looking at i.e 800.
+ *
+ * If you prefer a fixed field of view, follow these steps:
+ *
+ * 1.Choose your desired field of view angle (`fovy`). This is how wide the camera can see.
+ *
+ * 2.To position the camera correctly, use the formula:
+ *   cameraDistance = (height / 2) / tan(fovy / 2);
+ *   This ensures that you can see the entire width across horizontally and height across vertically at the fixed field of view.
+ *
+ * 3.Set the near value to  cameraDistance / 10 and the far value to  cameraDistance * 10 .
+ *
+ * 4.Simply, call perspective with the chosen field of view, canvas aspect ratio, and near/far values:
```

Comment:
Super tiny tweak, but can we take out "simply" at the start? We generally try to keep things instructional without passing judgment on how easy or hard things are.